src/main/java/seedu/homerce/logic/parser/ParserUtil.java

```diff
@@ -280,7 +282,7 @@ private static boolean isValidYear(String year) {
             return false;
         }
         int intYear = Integer.parseInt(year);
-        Year yearObject = Year.of(intYear);
+        Year yearObject = Year.of(VALID_YEAR);
         ValueRange range = yearObject.range(ChronoField.YEAR);
```

Comment:
I think this implementation still allows for `intYear` to be negative? For example `range.isValidIntValue(-999)` will still return true. Since the [java.time.Year class](https://docs.oracle.com/javase/8/docs/api/java/time/Year.html) has a `MAX_VALUE` of +999,999,999 and `MIN_VALUE` of -999,999,999.